python/setup.py

```diff
@@ -9,9 +9,9 @@
 
 packages = [p for p in find_packages() if "tests" not in p]
 
-PACKAGE_NAME = "snips_nlu_parsers"
+PACKAGE_NAME = "snips_nlu_parsers_genesys"
 ROOT_PATH = os.path.dirname(os.path.abspath(__file__))
-PACKAGE_PATH = os.path.join(ROOT_PATH, PACKAGE_NAME)
+PACKAGE_PATH = os.path.join(ROOT_PATH, "snips_nlu_parsers")
```

Comment:
I presume that package path is intended?

Reply:
Yes, the path is pointing to the python/snips_nlu_parsers directory. Package name is changed to reflect that we are forking the snips package